odl/diagnostics/operator.py

```diff
@@ -79,16 +82,19 @@ def norm(self):
 
         operator_norm = 0.0
```

Comment:
`operator_norm = max(self._opnorm_est(x) for _, x in samples(self.operator.domain))`